irohad/main/impl/pg_connection_init.hpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_PG_CONNECTION_INIT_HPP
+#define IROHA_PG_CONNECTION_INIT_HPP
+
+#include <soci/soci.h>
+
+#include <soci/callbacks.h>
+#include <soci/postgresql/soci-postgresql.h>
+#include <boost/algorithm/string.hpp>
+#include <boost/range/algorithm/replace_if.hpp>
+
+#include "ametsuchi/impl/failover_callback_holder.hpp"
+#include "ametsuchi/impl/pool_wrapper.hpp"
+#include "ametsuchi/impl/postgres_command_executor.hpp"
+#include "ametsuchi/impl/postgres_options.hpp"
+#include "ametsuchi/reconnection_strategy.hpp"
+#include "common/result.hpp"
+#include "interfaces/permissions.hpp"
+#include "logger/logger_fwd.hpp"
+#include "logger/logger_manager_fwd.hpp"
+
+namespace iroha {
+  namespace ametsuchi {
+    class PgConnectionInit {
+     public:
+      static std::string formatPostgresMessage(const char *message);
```

Comment:
Seems to be used only in `pg_connection_init.cpp`, maybe move it there in an anonymous namespace? Same for `processPqNotice`